tests/test_port_dpb_system.py

```diff
@@ -0,0 +1,909 @@
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode4x)
+        dvs.change_port_breakout_mode(rootPortName, breakoutMode1x, "-l")
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode4x)
+        dvs_acl.verify_acl_table_groups(0)
+        self.dvs_vlan.get_and_verify_vlan_member_ids(0)
+
+        # Delete ACL table, Add back VLAN table and
+        # ensure breakout WITH "-l" fails
+        dvs_acl.remove_acl_table(aclTableNames[0])
+        dvs_acl.verify_acl_table_count(0)
+        self.dvs_vlan.create_vlan(vlanIDs[0])
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode4x)
+        dvs.change_port_breakout_mode(rootPortName, breakoutMode1x, "-l")
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode4x)
+        dvs_acl.verify_acl_table_groups(0)
+        self.dvs_vlan.get_and_verify_vlan_member_ids(0)
+
+        # Add back ACL table and ensure, breakout succeeds
+        bind_ports = []
+        dvs_acl.create_acl_table(aclTableNames[0], "L3", bind_ports)
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode4x)
+        dvs.change_port_breakout_mode(rootPortName, breakoutMode1x, "-l")
+        dpb.verify_port_breakout_mode(dvs, rootPortName, breakoutMode1x)
+        dvs_acl.verify_acl_table_groups(1)
+        self.dvs_vlan.get_and_verify_vlan_member_ids(1)
+
+        # Delete ACL and VLAN tables
+        self.dvs_vlan.remove_vlan(vlanIDs[0])
+        dvs_acl.remove_acl_table(aclTableNames[0])
+
+        # TBD: Provide "-l" option without port_breakout_config_db.json file
+
+        # Verify cleanup
+        dvs_acl.verify_acl_table_count(0)
+        self.dvs_vlan.get_and_verify_vlan_ids(0)
+
+    def test_dpb_arp_flush(self, dvs):
+        dvs.setup_db()
+        self.setup_db(dvs);
+
+        portName = "Ethernet0"
+        vrfName = ""
+        ipAddress = "10.0.0.0/31"
+        srv0MAC = "00:00:00:00:01:11"
+
+        self.clear_srv_config(dvs)
+
+        # Create l3 interface
+        rif_oid = self.create_l3_intf(portName, vrfName)
+
+        # set ip address
+        self.add_ip_address(portName, ipAddress)
+
+        # bring up interface
+        self.set_admin_status(dvs, portName, "up")
+
+        # Set IP address and default route
+        cmd = "ip link set eth0 address " + srv0MAC
+        dvs.servers[0].runcmd(cmd)
+        dvs.servers[0].runcmd("ip address add 10.0.0.1/31 dev eth0")
+        dvs.servers[0].runcmd("ip route add default via 10.0.0.0")
+
+        # Get neighbor and ARP entry
+        dvs.servers[0].runcmd("ping -c 1 10.0.0.0")
+
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_NEIGHBOR_ENTRY")
+        intf_entries = tbl.getKeys()
+        assert len(intf_entries) == 1
+        route = json.loads(intf_entries[0])
+        assert route["ip"] == "10.0.0.1"
+        assert route["rif"] == rif_oid
+        (status, fvs) = tbl.get(intf_entries[0])
+        assert status == True
```

Comment:
`wait_for_result` would help here

Reply:
Instead, I am using wait_for_n_keys() and wait_for_exact_match() methods to ensure neighbor entry is present/learned